services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/api_client/_public.py

```diff
@@ -81,7 +81,7 @@ def _dynamic_services_scheduler_settings(self) -> DynamicServicesSchedulerSettin
         return settings
 
     async def is_healthy(
-        self, dynamic_sidecar_endpoint: AnyHttpUrl, *, with_retry: bool = True
+        self, dynamic_sidecar_endpoint: str, *, with_retry: bool = True
```

Comment:
this is wrong, in the entire file. Please revert it.

Reply:
done